README.md

```diff
@@ -84,6 +84,23 @@ If you would prefer to write it into a file or pipe it into another application,
 youtube_transcript_api <first_video_id> <second_video_id> ... --languages de en --json > transcripts.json
 ```
 
+### Proxy
+
+You can pass a proxy to use during the network requests
+
+Code:
+```python
+from youtube_transcript_api import YouTubeTranscriptApi
+
+YouTubeTranscriptApi.get_transcript(video_id, proxy={"http": "http://user:pass@domain:port", "https": "https://user:pass@domain:port"})
+
+```
+
+CLI:
+```
+youtube_transcript_api <first_video_id> <second_video_id> http://user:pass@domain:port
```

Comment:
```suggestion
youtube_transcript_api <first_video_id> <second_video_id> --proxy http://user:pass@domain:port
```

Reply:
The CLI doesn't allow for specifying individual proxies for HTTP and HTTPS and will implicitly use the HTTP proxy on HTTPS. I don't know if this will be a problem for your use case, but maybe you should consider using a `--http-proxy` and `--https-proxy` param.